.github/workflows/tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       max-parallel: 2
       matrix:
         python-version: ["3.8"]
```

Comment:
I think it's important to add test support for Python3.10 in the test.yml, setup.py, and tox.ini to follow the [Support Windows](https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit?usp=sharing)

What do you think?

Reply:
@MaferMazu actually openedx-events is not working with Python3.10

![image](https://user-images.githubusercontent.com/39854568/194185280-ad01638d-c69d-4d4a-88b9-73f5417ab2ab.png)

Reply:
Ok, ok, thanks for the info.